sbin/build.sh

```diff
@@ -153,7 +153,11 @@ configuringVersionStringParameter()
   local openJdkVersion=$(getOpenJdkVersion)
   echo "OpenJDK repo tag is ${openJdkVersion}"
 
-  addConfigureArg "--with-milestone=" "fcs"
+  # --with-milestone=fcs deprecated at jdk11, removed at jdk12
+  if [ "${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" != "${JDK12_CORE_VERSION}" ]; then
```

Comment:
It's also true for JDKHEAD I assume?

Reply:
Probably a fair assumption.  I'll change it.

Reply:
Excluded --with-milestone=fcs from HEAD builds